docs/workspaces/index.md

```diff
@@ -274,3 +274,18 @@ Instructors are strongly encouraged to allow students to get exposed to the Prai
 - Workspace-specific instructions. Different workspace environments may include different instructions for saving files, compiling, testing, and otherwise using the environment, as well as different directory structures that may be relevant in some contexts. Additionally, some workspaces may require additional actions in specific cases. For example, in Jupyter workspaces, students may need to know how to interrupt or restart the kernel.
 
 - Expectations about workspace start times. In the worst case, a workspace may take several minutes as new machines are started and as images are pulled. This may be especially important in timed assessments, where the workspace starting time may affect the student's ability to complete their work.
+
+## Troubleshooting workspace loading
+
+A variety of issues can prevent a workspace from loading, including antivirus software, browser extensions, and network issues. If a workspace fails to load, the following steps may help you resolve the issue or provide more information to the PrairieLearn team:
+
+- **Wait**: Sometimes workspaces take a few minutes to start, especially if the workspace image is being pulled for the first time. Even once a workspace's state changes to running, slow networks may result in a blank page for some time as the workspace's client-side assets load.
```

Comment:
Maybe emphasize that there is a percentage counter and that it's important to wait for this to reach 100?

Reply:
How do the changes in 1937189 look?